capstone/src/main/java/com/google/sps/gmail/EmailFactory.java

```diff
@@ -80,10 +81,21 @@ private static void sendEmail(String recipientEmail, String body, String subject
 
   private static String getHTMLAsString(String path) throws IOException {
     // Load HTML file and convert to String
-    String fullPath = Constants.EMAIL_PATH + path;
-    File htmlTemplate = new File(fullPath);
-    String emailBody = FileUtils.readFileToString(htmlTemplate, "utf-8");
-    return emailBody;
+    InputStream is = EmailFactory.class.getResourceAsStream(Constants.EMAIL_PATH + path);
+    ByteSource byteSource =
+        new ByteSource() {
+          @Override
+          public InputStream openStream() throws IOException {
+            return is;
+          }
+        };
+
+    String text = byteSource.asCharSource(Charsets.UTF_8).read();
+    return text;
+    // String fullPath = new File(".").getCanonicalPath() + Constants.EMAIL_PATH + path;
+    // File htmlTemplate = new File(fullPath);
+    // String emailBody = FileUtils.readFileToString(htmlTemplate, "utf-8");
+    // return emailBody;
```

Comment:
nit: remove commented code
Could we also change the variables names to something more understandable so is -> inputStream and text -> emailBody?